SUMMARY.md

```diff
@@ -105,72 +104,28 @@
-  * [Using data elements to define dynamic data in rules](resources/user-guides/launch-data-elements.md)
-  * [Signal extension and Rules Engine integration](resources/user-guides/signal-extension-and-rules-engine-integration.md)
-* [Troubleshooting Guides](resources/troubleshooting-guides/README.md)
-  * [Push Messaging](resources/troubleshooting-guides/troubleshooting-push.md)
-  * [Campaign Push Notifications](resources/troubleshooting-guides/push_notification_troubleshooting.md)
-* [Building mobile extensions](resources/building-mobile-extensions/README.md)
-  * [Event processing](resources/building-mobile-extensions/event-processing.md)
-  * [Shared states and events](resources/building-mobile-extensions/shared-states-and-events.md)
-  * [Building an extension](resources/building-mobile-extensions/building-an-extension.md)
-  * [Listening for events](resources/building-mobile-extensions/event-listeners.md)
-  * [Dispatching events from your extension](resources/building-mobile-extensions/dispatching-events-from-your-extension.md)
-  * [Requesting a shared state](resources/building-mobile-extensions/requesting-a-shared-state.md)
-  * [Requesting a shared state that is not tied to an event](resources/building-mobile-extensions/requesting-a-shared-state-that-is-not-tied-to-an-event.md)
-  * [Updating the shared state](resources/building-mobile-extensions/updating-the-shared-state.md)
-  * [Updating a shared state that is not tied to an event](resources/building-mobile-extensions/updating-a-shared-state-that-is-not-tied-to-an-event.md)
-  * [Monitoring shared states](resources/building-mobile-extensions/monitoring-shared-states.md)
-  * [Clearing a shared state](resources/building-mobile-extensions/clearing-a-shared-state.md)
-  * [Building the Experience Platform Launch extension interface](resources/building-mobile-extensions/building-the-launch-extension-interface.md)
-  * [Interacting with rules](resources/building-mobile-extensions/interacting-with-rules.md)
-  * [Get the application context](resources/building-mobile-extensions/get-the-application-context.md)
-  * [Extension logging](resources/building-mobile-extensions/logging-from-extension.md)
-  * [Mobile extension samples](resources/building-mobile-extensions/sample-extensions.md)
 * [Frequently asked questions](resources/frequently-asked-questions/README.md)
   * [Latest SDK Versions](https://aep-sdks.gitbook.io/docs/resources/upgrading-to-aep/current-sdk-versions)
 * [Privacy and GDPR](resources/privacy-and-gdpr.md)
-* [WeChat Mini Programs](resources/adobe-experience-platform-mini-programs-sdk/README.md)
-  * [Implementation](resources/adobe-experience-platform-mini-programs-sdk/implementation.md)
-  * [Implementation - 中文–简体](resources/adobe-experience-platform-mini-programs-sdk/implementation-zhong-wen-1.md)
-  * [Frequently Asked Questions](resources/adobe-experience-platform-mini-programs-sdk/faq.md)
 
 ## BETA
```

Comment:
should we remove this beta section too? we have a section for Assurance under Foundation extensions, I think we should keep any tutorials in the main branch only.

Reply:
Yes we should, I wasn't sure if it was being done as part of one of the other PRs. I can make the updates here.

Reply:
👍 Cool, not covered in my PRs at this point, so yup let's remove that too.